Elmah.Io.ElasticSearch/ElasticClientSingleton.cs

```diff
@@ -18,9 +18,13 @@ public class ElasticClientSingleton : IDisposable
         private static ElasticClientSingleton _instance;
         public IElasticClient Client;
 
-        private ElasticClientSingleton()
+        private ElasticClientSingleton(IDictionary config = null)
         {
-            var config = ReadConfig();
+            if(ReferenceEquals(config, null))

```

Comment:
@jayhilden usually I always use this way to compare references to avoid any operator overload